Assets/Python/RFCUtils.py

```diff
@@ -172,16 +172,16 @@ def isMortalUnit(self, unit):
 				if (gc.getGame().getSorenRandNum(100, 'random modifier') >= 50):
 					return False	      
 		iUnitType = unit.getUnitType()
-		if (iUnitType <= con.iKhmerBallistaElephant \

```

Comment:
Good catch, but this method is a mess in general. I pushed a much cleaner implementation to develop, you might need to rebase though. Sorry about that.